src/Repository/ResearchGroupRepository.php

```diff
@@ -81,4 +81,20 @@ public function getResearchGroupsInfo(array $aggregations): array
 
         return $researchGroupsInfo;
     }
+
+    /**
+     * Find the lowest-value ID available for use, in a specified range.
+     *
+     * @param int $min Range start.
+     * @param int $max Range end.
+     * @return int First available ID.
+     */
+    public function getNextAvailableId($min, $max)
+    {
+        $researchGroups = $this->findAll();
```

Comment:
Here use the querybuilder, then only select ID. In results use Query::Hydrate_Array) and you'll have your array.

Reply:
QUERY::HYDRATE_SCALAR_COLUMN seemed to be the magic.